# 8. What are custom exceptions?
#
# Subclass Exception:

class MyError(Exception): pass